content/applications/websites/website/web_design/building_blocks/features_blocks.rst

```diff
@@ -0,0 +1,28 @@
+===============
+Features blocks
+===============
+
+Use the :guilabel:`Features` blocks to add advanced functionalities to your website, such as
+describing the attributes of a product or service. The blocks from this category allow you to
+present the content of your website in an illustrative method rather than sticking to the textual
+content. This documentation presents the :guilabel:`Table of content` building block.
+
+.. image::features_blocks/features_blocks.png
+   :alt: The features blocks available in the website builder.
+
+.. seealso::
+   :doc:`Building blocks <../building_blocks>`
+
+Table of content
+================
+
+You can add a :guilabel:`Table of content` building block to your website by dragging and dropping
```

Comment:
I would link this page here so they can know the more general setting on how to add blocks https://runbot137.odoo.com/runbot/static/build/59908338-17-0/logs/build/html/applications/websites/website/web_design/building_blocks.html